website/snippets/_packages_or_dependencies.md

```diff
@@ -2,7 +2,9 @@
 ## Use cases
 
 Starting from dbt v1.6, `dependencies.yml` has replaced `packages.yml`. The `dependencies.yml` file can now contain both types of dependencies: "package" and "project" dependencies.
-- ["Package" dependencies](/docs/build/packages) lets you add source code from someone else's dbt project into your own, like a library. 
+- ["Package" dependencies](/docs/build/packages) lets you add source code from someone else's dbt project into your own, like a library.
+
+  [comment]: Why add a link to this bulletpoint that redirects the reader to the same website? It makes no sense because it's a never ending loop. It's like having a website named "Make a meal", that has a list of bulletpoints like: "1- Buy it from a restaurant. 2- Make a meal" and the second bulletpoint sends the reader to itself. My proposal is to remove this looping link. Regards.
```

Comment:
great flag @ialdg and i think this is a reusable snippet that we use in other pages (example [here](https://docs.getdbt.com/docs/collaborate/govern/project-dependencies#use-cases)) but you're right inthat we can be a bit more targeted and not send the reader through this weird loop. I've updated the link to a specific header so the read is taken somewhere with purpose in either page. let me know if that works for you!